### PR TITLE
HW_16

### PR DESCRIPTION
**Pull request by Tetiana-Kulyk**

### CODE REVIEW

File: HW_16/HW_16.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+website = "https://djinni.co/"
+
+# Ancestor
+xpath_selector_1 = "//a[@data-analytics-param='login_homepage']/ancestor::li"
```
Comment (bohdandovbsyh) [inline HW_16/HW_16.py:4]: image

File: HW_16/HW_16.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+xpath_selector_2 = "//a[@class='navbar-brand']/ancestor-or-self::a"
+
+# Attribute
+xpath_selector_3 = "//attribute::target"
+css_selector_1 = "[target]"
+
+xpath_selector_4 = "//a/attribute::rel"
+
+# Child
+xpath_selector_5 = "/html/body/div[1]/div[1]/div[1]/h1"
```
Comment (bohdandovbsyh) [inline HW_16/HW_16.py:16]: image

File: HW_16/HW_16.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+xpath_selector_6 = "//child::h1"
+
+xpath_selector_7 = "//div[@class='container container_landing']/child::p"
+css_selector_2 = "p:only-child"
+
+# Descendant
+xpath_selector_8 = "//div[@align='center']/descendant::a"
+
+# Descendant-of-self
+xpath_selector_9 = "//div[@class='jobs-push-briefly']/descendant-or-self::p"
```
Comment (bohdandovbsyh) [inline HW_16/HW_16.py:26]: image


I just open "https://djinni.co/" and cant find lots of locators. Do I nee some preconditions to make it works?

Comment (Tetiana-Kulyk) [reply HW_16/HW_16.py:26]: image

File: HW_16/HW_16.py (hunk 2 above)
Comment (Tetiana-Kulyk) [reply HW_16/HW_16.py:16]: image

File: HW_16/HW_16.py (hunk 1 above)
Comment (Tetiana-Kulyk) [reply HW_16/HW_16.py:4]: image

Comment (bohdandovbsyh) [reply HW_16/HW_16.py:4]: Looks like for English version of this site locators are different.

**Pull request merged**